mne/transforms.py

```diff
@@ -63,8 +63,6 @@ def _to_const(cf):
         if cf not in _str_to_frame:
             raise ValueError('Unknown cf %s' % cf)
         cf = _str_to_frame[cf]
-    elif not isinstance(cf, (int, np.integer)):
-        raise TypeError('cf must be str or int, not %s' % type(cf))
     return int(cf)
```

Comment:
I think it prevents people from accidentally passing `float`. The correct thing to do, though, is really to check against `from numbers import Integral; if isinstance(x, Integral)`